modules/oe_theme_content_project/src/Plugin/ExtraField/Display/FundingExtraField.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_project\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+
+/**
+ * Extra field displaying Funding and Proposals Project fields.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_project_funding",
+ *   label = @Translation("Funding"),
+ *   bundles = {
+ *     "node.oe_project",
+ *   },
+ *   visible = true
+ * )
+ */
+class FundingExtraField extends ExtraFieldDisplayFormattedBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(ContentEntityInterface $entity) {
+    $funding = $entity->get('oe_project_funding_programme')->getIterator();
+    foreach ($funding as $item) {
+      $build['#items'][] = [
+        'fieldlabel' => $entity->oe_project_funding_programme->getFieldDefinition()->getLabel(),
+        'link' => [
+          'text' => $item->entity->pref_label->value,
+          'url' => $item->target_id,
+        ],
+      ];
+    }
+    $calls = $entity->get('oe_project_calls')->getIterator();
+    foreach ($calls as $item) {
+      $build['#items'][] = [
+        'fieldlabel' => $entity->oe_project_calls->getFieldDefinition()->getLabel(),
+        'link' => [
+          'text' => $item->title,
+          'url' => $item->uri,
+        ],
+      ];
+    }
+
+    return $build;
```

Comment:
Let's use the list item pattern, default variant.

Reply:
Changed to use list item pattern